subsys/bluetooth/host/direction.c

```diff
@@ -689,11 +689,13 @@ int hci_df_prepare_connection_iq_report(struct net_buf *buf,
 
 	if (!atomic_test_bit(conn->flags, BT_CONN_CTE_RX_ENABLED)) {
 		LOG_ERR("Received conn CTE report when CTE receive disabled");
+		bt_conn_unref(conn);
```

Comment:
```suggestion
		bt_conn_unref(conn);
		
		return;
```
:p

Reply:
Is this a general comment for all the cases, or just this specific one?

Reply:
all the cases, mostly tongue-in-cheek since it's a rule you're trying to follow (and make us follow). Im not blocking the PR at all :)

Reply:
True :D I can update that no problem